backend/src/dfini_hack_team4.did

```diff
@@ -0,0 +1,59 @@
+type Amount = nat64;
+type AnswerId = nat64;
+type Solution = blob;
```

Comment:
just to be consistent, should we say `answerId` and `answer` or `solutionId` and `solution`?

Reply:
I think I'll go with `Content` instead of `Solution` because it captures the intent better and keep `Answer` around.